Sources/BraveWallet/Crypto/CryptoPagesView.swift

```diff
@@ -83,6 +83,7 @@ struct CryptoPagesView: View {
         .sheet(isPresented: $cryptoStore.isPresentingAssetSearch) {
           AssetSearchView(
             keyringStore: keyringStore,
+            networkStore: cryptoStore.networkStore,
             cryptoStore: cryptoStore,
             userAssetsStore: cryptoStore.portfolioStore.userAssetsStore
```

Comment:
can define the init to only take `keyringStore` and `cryptoStore`? 
init(keyringStore: KeyringStore, cryptoStore: CryptoStore) {
  self.keyringStore = keyringStore
  self.networkStore = cryptoStore.networkStore
  self.cryptoStore = cryptoStore
  self.userAssetStore = cryptoStore.portfolioStore.userAssetStore
}

Reply:
Addressed in [85b7dfe](https://github.com/brave/brave-ios/pull/7617/commits/85b7dfea49074c11a09f7d3736e6705468eff44b).